content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -1,7 +1,137 @@
 ---
-title: "Working with Kubernetes Objects"
-weight: 40
+title: Objects In Kubernetes
+content_type: concept
+weight: 10
 description: >
-  Kubernetes objects are persistent entities in the Kubernetes system. Kubernetes uses these entities to represent the state of your cluster.
+  Kubernetes objects are persistent entities in the Kubernetes system.
+  Kubernetes uses these entities to represent the state of your cluster.
   Learn about the Kubernetes object model and how to work with these objects.
+simple_list: true
+card:
+  name: concepts
+  weight: 40
 ---
+
+<!-- overview -->
+
+This page explains how Kubernetes objects are represented in the Kubernetes API, and how you can
+express them in `.yaml` format.
+
+<!-- body -->
+
+## Understanding Kubernetes objects {#kubernetes-objects}
+
+*Kubernetes objects* are persistent entities in the Kubernetes system. Kubernetes uses these
+entities to represent the state of your cluster. Specifically, they can describe:
+
+* What containerized applications are running (and on which nodes)
+* The resources available to those applications
+* The policies around how those applications behave, such as restart policies, upgrades, and fault-tolerance
+
+A Kubernetes object is a "record of intent"--once you create the object, the Kubernetes system
+will constantly work to ensure that object exists. By creating an object, you're effectively
+telling the Kubernetes system what you want your cluster's workload to look like; this is your
+cluster's *desired state*.
+
+To work with Kubernetes objects--whether to create, modify, or delete them--you'll need to use the
+[Kubernetes API](/docs/concepts/overview/kubernetes-api/). When you use the `kubectl` command-line
+interface, for example, the CLI makes the necessary Kubernetes API calls for you. You can also use
+the Kubernetes API directly in your own programs using one of the
+[Client Libraries](/docs/reference/using-api/client-libraries/).
+
+### Object spec and status
+
+Almost every Kubernetes object includes two nested object fields that govern
+the object's configuration: the object *`spec`* and the object *`status`*.
+For objects that have a `spec`, you have to set this when you create the object,
+providing a description of the characteristics you want the resource to have:
+its _desired state_.
+
+The `status` describes the _current state_ of the object, supplied and updated
+by the Kubernetes system and its components. The Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} continually
+and actively manages every object's actual state to match the desired state you
+supplied.
+
+For example: in Kubernetes, a Deployment is an object that can represent an
```

Comment:
For example: a Deployment is an object ... application running on your Kubernetes cluster.

Reply:
I don't understand the feedback here. Can you frame your thoughts as a GitHub suggestion @kbhawkey ?